README.md

```diff
@@ -1,12 +1,11 @@
-# bls-js
+# bls
 
-[![Build Status](https://travis-ci.org/ChainSafe/lodestar.svg?branch=master)](https://travis-ci.org/ChainSafe/lodestar)
-[![codecov](https://codecov.io/gh/ChainSafe/lodestar/branch/master/graph/badge.svg)](https://codecov.io/gh/ChainSafe/lodestar)
```

Comment:
Why not add updated pills?

Reply:
We don't use travis anymore
But I was feeling lazy on codecov, will update